spec/13-syntax-summary.md

```diff
@@ -132,7 +132,7 @@ grammar:
   Expr1             ::=  `if' `(' Expr `)' {nl} Expr [[semi] `else' Expr]
                       |  `while' `(' Expr `)' {nl} Expr
                       |  `try' (`{' Block `}' | Expr) [`catch' `{' CaseClauses `}'] [`finally' Expr]
-                      |  `do' Expr [semi] `while' `(' Expr ')'
+                      |  `do' Expr [semi] `while' `(' Expr `)'
```

Comment:
The doc is informal, so arguably typos don't matter. There are already two quoting styles tho, and having a third just for this bracket seems excessive (what could be so special about *this* bracket?)